onyx/onyx_sram_subsystem/mock_mem.py

```diff
@@ -0,0 +1,328 @@
+import magma as m
+
+# I am not making the read latency a generator param
+# as it is not parameter of Chisel provided by Gedeon
+READ_LATENCY: int = 1
```

Comment:
Should this be 0 or does the SRAM have a registered output?

Reply:
Actually now that I think about the power gating options with the preservation of the Q, it may be that the output is registered